### ISSUE TITLE
Avoid RPC errors sourced from us

### ISSUE THREAD

**Issue by SabaunT** [gear-tech/gear#1499]
### Problem

After #1498 the `node-loader` sometimes receives an error (see log output) from the node. Need to investigate the reasons and apply fixes so an error is not born on the tool's side. 

### Steps

-

### Relevant Log Output

<details><summary>Click to expand/collapse</summary>
<p>

```
ERROR: Rpc error: RPC call failed: ErrorObject { code: ServerError(1014), message: "Priority is too low: (236 vs 212)", data: Some(RawValue("The transaction has too low priority to replace another transaction already in the pool.")) }
```

</p>
</details>

**Comment by SabaunT** [comment]
That was re-producable by `node-loader`. @breathx fixes this in his crate. The issue is no longer actual
